src/main/java/com/qunite/api/QuniteApiApplication.java

```diff
@@ -1,7 +1,14 @@
 package com.qunite.api;
 
+import java.util.Arrays;
+
+import org.springframework.boot.CommandLineRunner;
```

Comment:
⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'org.springframework.boot.CommandLineRunner'